lib/view_helpers.rb

```diff
@@ -4,5 +4,26 @@ def input_userattr(user, attr, opt = {})
       opt[:class] ||= 'form-control'
       haml_tag :input, {name: attr, value: user[attr].first}.merge(opt)
     end
+    def select_userattr(user, attr, values, showDefault, opt = {})
+      opt[:class] ||= 'form-select'
+      haml_tag :select, {name: attr}.merge(opt) do
+        if (showDefault) then
```

Comment:
Rubyでは変数名はsnake_caseにするのが一般的なので揃えてほしいです
またifに括弧はいらないです（改行する場合はthenもいらないです）
```suggestion
    def select_userattr(user, attr, values, show_default, opt = {})
      opt[:class] ||= 'form-select'
      haml_tag :select, {name: attr}.merge(opt) do
        if show_default
```